tests/models.py

```diff
@@ -369,8 +369,13 @@ class StatusFieldChoicesName(models.Model):
 
 
 class StatusFieldChoicesDisplay(models.Model):
-    NAMED_STATUS = (("UNCB", _("Unconfirmed Booking")), ("CONB", _("Confirmed Booking")))
-    status = StatusField(choices_name='NAMED_STATUS')
+    UNCONFIRMED_BOOKING = 'UNCB'
+    CONFIRMED_BOOKING = 'CONB'
+    JOB_STATUSES = ((UNCONFIRMED_BOOKING, _("Unconfirmed Booking")),
+                    (CONFIRMED_BOOKING, _("Confirmed Booking"))
+                    )
+    STATUS = JOB_STATUSES
+    status = StatusField(default=CONFIRMED_BOOKING)
```

Comment:
any migration change needed in test?

Reply:
Good point. I'm not sure, to be completely honest with you. I am interested if the type of quotes used for the definitions matters at all? i.e. I'm using `CONFIRMED_BOOKING = 'CONB'` and not `CONFIRMED_BOOKING = "CONB"`, if that might have an effect?
I'm also using sqlite in testing locally, but postgresql in production.